packages/twenty-front/src/pages/object-record/__stories__/RecordIndexPage.stories.tsx

```diff
@@ -44,6 +46,8 @@ export const Default: Story = {
 };
 
 export const Loading: Story = {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   decorators: [PageDecorator],
```

Comment:
For some reason, a decorator with typed `args` will throw an error if placed in a `Story`